src/errorpage.cc

```diff
@@ -1063,7 +1139,15 @@ ErrorState::Convert(char token, bool building_deny_info_url, bool allowRecursion
         break;
 
     default:
-        mb.appendf("%%%c", token);
+        if (building_deny_info_url && errorHandler_) {
+            try {
+                // catch the exceptions we need only to report the error.
+                errorHandler_->handleError(ToSBuf("Wrong error code at  '%", SBuf(start).substr(0, 100), "'"));
+            } catch(...) {
+                // do nothing, errors printed inside errorHandler
+            }
```

Comment:
This looks strange: basically, errorHandler_->handleError() purpose may not include reporting error messages. Is it possible to eliminate this confusing try/catch_all block, with a new ErrorHandler::reportError() method?

Reply:
There is and used  the new ErrorHandler::report method now.